src/core/lib/channel/tcp_tracer/BUILD

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bazel:grpc_build_system.bzl",
```

Comment:
We want to remove these two and use cc_library and not grpc_cc_library, or something equivalent.

Reply:
I think we can add an exception to prod `grpc_build_system.bzl` on top of this change. I confirmed the build overhead increase becomes minimal when I added a hard-coded exception, similar to what we have for `eventmanager_interface`.